### ISSUE TITLE
orion-ld don´t connect to mongodb community operator 

### ISSUE THREAD

**Issue by fabiocabrini** [Fiware/context.Orion-LD#1447]
I am attempting to connect Orion-LD to the MongoDB Community Operator in a Kubernetes (k8s) environment, but I am encountering the following error:
```
time=Friday 06 Oct 15:25:17 2023.339Z | lvl=WARN | corr=N/A | trans=N/A | from=N/A | srv=N/A | subsrv=N/A | comp=Orion | op=mongocInit.cpp[58]:mongocLog | msg=MONGOC[mongoc]: A '/' is required between the host list and any options.
time=Friday 06 Oct 15:25:17 2023.339Z | lvl=FATAL | corr=N/A | trans=N/A | from=N/A | srv=N/A | subsrv=N/A | comp=Orion | op=mongocInit.cpp[294]:mongocInit | msg=Unable to connect to mongo(URI: mongodb://my-user:xxxx@orionld-mongodb-svc:27017?replicaSet=orionld-mongodb&authMechanism=SCRAM-SHA-256&): Invalid host string in URI
```
Interestingly, when I use the standard Orion, this issue does not occur. In my view, there is an error in the connection string because a '/' is required between the host list and any options.

Could someone please assist me with this issue?

Here's the relevant part of my Kubernetes configuration:

```yaml
spec:
  containers:
  - name: orion
    image: fiware/orion-ld
    args: ["-dbhost","orionld-mongodb-svc:27017","-rplSet","orionld-mongodb","-dbuser","my-user","-dbpwd","xxxx","-dbAuthMech","SCRAM-SHA-256","-db","orion","-logLevel","INFO"]
    ports:
    - containerPort: 1026
```

**Comment by fabiocabrini** [comment]
The issue was resolved using the following parameters:
`args: ["-dbURI", "mongodb+srv://my-user:xxxx@orionld-mongodb-svc.default.svc.cluster.local/admin?replicaSet=orionld-mongodb&authMechanism=SCRAM-SHA-256&ssl=false"]`

**Comment by kzangeli** [comment]
Sorry for not responding. Out on vacation. Back today.
While using -dbURI is a perfect "workaround" (not really a workaround), you've encountered a bug, one that I've fixedf already but that is still not merged due to problems with github actions. I', planning on merging the PR today even though Orion-LD's github actions is still not working

**Comment by kzangeli** [comment]
Sorry, forgot to answer you here.
I got the github actions fixed and I merged the PR some time ago.
Could you test again and let me know? (by closing the issue if all OK or adding a new comment if not)

**Comment by fabiocabrini** [comment]
Hi Ken, I’m so sorry for the inconvenience. The issue has been solved. Here is the solution:
`args: ["-dbURI", "mongodb+srv://my-user:xxxxxx@orionld-mongodb-svc.default.svc.cluster.local/admin?replicaSet=orionld-mongodb&authMechanism=SCRAM-SHA-256&ssl=false"]`

I have added this code to my .yaml:

`env:
        - name: ORIONLD_MONGOCONLY
          value: "TRUE"`
       
The Orion-LD connected to the MongoDB Community Operator perfectly.
Thanks!
